scripts/qa_crowbarsetup.sh

```diff
@@ -3338,12 +3338,14 @@ function oncontroller_run_tempest
 
     if iscloudver 6plus; then
         tempest cleanup --init-saved-state
-        local opts=$tempestoptions
+        local opts=($tempestoptions)
         if iscloudver 7plus; then
             ### backward compatibility, remove
-            opts=${opts/-s/--smoke}
-            opts=${opts/-t/--serial}
-            tempest run $opts 2>&1 | tee tempest.log
+            for i in ${!opts[@]}; do
+                opts[$i]=${opts[$i]/-s/--smoke}
```

Comment:
Hmm, I was trying to avoid that by splitting on whitespace in line 3341. You don't think that's going to work?